html2/tokenizer_test.cpp

```diff
@@ -458,5 +458,34 @@ int main() {
         expect_eq(glyph, "∾̳"sv);
     });
 
+    etest::test("attribute, one attribute single qouted", [] {
+        auto tokens = run_tokenizer("<tag a=\'b\'>");
```

Comment:
You don't have to escape `'` inside `"` strings.

Reply:
Fixed!